docs/testplan/Smartswitch-test-plan.md

```diff
@@ -384,7 +411,7 @@ root@sonic:/home/cisco# show chassis modules status
  * Verify DPU is shown in show chassis modules status after DPU powered on
 
 
-### 1.7 Check removal of pcie link between NPU and DPU
+### 1.7 Check pcie link status between NPU and DPU
```

Comment:
Nit: Optional, using "Verify" in place of "Check" in each of the test header provides most consistent wording.

Reply:
Actually, I consistently followed "Check" in all the test cases.

Can we leave it as such to maintain it?

Reply:
okay.